Makefile

```diff
@@ -11,12 +11,14 @@ run-%-local:
 	./scripts/wait.sh 
 	docker-compose -f docker-compose.$*.yaml up --no-build -d 
 	./scripts/wait.sh
+	docker ps -a
```

Comment:
the wait script already does `docker ps -a`

Reply:
Yes it does, but not printing in logs as a command output
![image](https://user-images.githubusercontent.com/72209717/161950778-4135b46a-5b94-40a4-8ca8-a8a655ef6e3b.png)
![image](https://user-images.githubusercontent.com/72209717/161951036-3513adb0-c61e-4c30-b3cf-d5d78ab81bb2.png)

I can add `echo docker ps -a` into wait script after the  `echo "All services are healthy"`

Reply:
It seems that this is for debugging only. If everything works, it should be not printed for the end-user.
Can we print it only for Jenkins?

Reply:
OK
I'll move it into the Jenkinsfile

Reply:
I've added a DEBUG environment variable to `Jenkinsfile` and I've updated `wait.sh` script and `make clean` operation.